data/hai/hai reveal 2 start.txt

```diff
@@ -0,0 +1,368 @@
+					goto decline
+			`	Are you sure? Pirates will chase you. I'd rather not die on your ship.`
+			choice
+				`	"Don't worry. I know how to dodge pirates."`
+				`	"You should find a different ride then."`
+					decline
+			`	"Okay. I guess I should have introduced myself... it's easy to forget social niceties in a place like this. My name is Ooonem, and I'm a researcher from <planet>. Thank you for agreeing to take me home."`
+			`	You notice several men nearby looking intently at Ooonem. The two of you run towards your ship and make it to cover before the men start shooting. Ooonem pulls a grenade out of her cloak and throws it at the men. Gas explodes outward and the men drop dead in seconds.`
+			`	"I had better leave these behind," she says, dropping a few gas grenades on the ground. "I took them off of the pirates a few days ago. They're quite illegal." You notice a label on the grenades "tabun gas: twenty-meter radius." This was a nerve gas grenade, using one of the older varieties of the gas. Cheap and plentiful in pirate space.`
+			`	You run into your ship and immediately take off.`
+				launch
+			label decline
+			`	The woman runs off into the shadows. You doubt you'll ever see her again.`
+				decline
+	on complete
+		log `Rescued a kidnapped Hai researcher, Ooonem, from Stormhold and returned her to Makerplace. She was tricked into leaving Hai space for a fake business deal, and the merchant who agreed to take her there kidnapped her instead. Men have been hunting her since she escaped. She doesn't know the real identities of the kidnappers.`
+		payment 225000
+		"hai slave prereq" ++
+		dialog `Ooonem is relieved to return to <planet>. She leads you to her family's home where you're treated to a bountiful dinner. The cuisine is entirely human-Hai fusion dishes. The dessert is accompanied by most delicious spice of all: <payment>.`
+	npc
+		government "Bounty Hunter that Won't Enter Hai Space"
+		personality plunders disables unconstrained
+		fleet "Large Core Pirates" 1
+		fleet "Small Core Pirates" 1
+
+
+mission "Hai Rescue: Meroot and Lucy"
+	landing
+	name "Return Meroot and Lucy"
+	description "Transport the scientists you rescued to <destination>. Payment is <payment>."
+	source Canyon
+	destination Newhome
+	passengers 2
+	repeat 3
+	to offer
+		"combat rating" > 50
+		has "Unwanted Cargo: done"
+		not "event: hai-human treaty signed"
+		not "Hai Rescue: Meroot and Lucy: offered"
+		"Hai Rescue: Meroot and Lucy (offer count)" < 3
+	on offer
+		"Hai Rescue: Meroot and Lucy (offer count)" ++
+		conversation
+			`As you walk through a major street, you notice a small alcove in the side of one building. Your sharp pilot eyes spot a woman within, with tufts of hair across her body. It's probably a Hai. Another woman, clearly human, is walking nearby carrying some food. She looks around to see if she's being followed before entering the alcove and covering it with a sheet of steel.`
```

Comment:
Might be better worded as "She's probably a Hai" instead of "It's probably a Hai", to make it more in line with the previous sentence.